e2e-tests/upgrade/run

```diff
@@ -62,34 +69,29 @@ function create_or_upgrade_to_version() {
 
 	kubectl_bin patch "perconapgcluster/${cluster}" --type json -p \
 		'[
-			{"op":"replace","path":"/spec/pause","value":true},
 			{"op":"replace","path":"/spec/pgBouncer/size","value":0},
 		]'
+	sleep 10
+	kubectl_bin patch "perconapgcluster/${cluster}" --type json -p \
+		'[
+			{"op":"replace","path":"/spec/pause","value":true},
+		]'
 	wait_cluster_status "${cluster}" "pgcluster Shutdown"
-
-	kubectl_bin delete \
-		serviceaccounts/pgo-deployer-sa \
-		clusterroles/pgo-deployer-cr \
-		configmaps/pgo-deployer-cm \
-		configmaps/pgo-config \
-		clusterrolebindings/pgo-deployer-crb \
-		jobs.batch/pgo-deploy \
-		deployment/postgres-operator \
-		deployment/some-name-pgbouncer \
-		service/some-name-pgbouncer || :
-
 }
 
 main() {
 	create_namespace $namespace
 	cluster="some-name"
 
+	operator_action='install'
 	if [[ ${PG_VER} -le '13' ]]; then
-		create_or_upgrade_to_version ${cluster} '0.2.0' "${test_dir}/conf/0.2.0_operator.yaml"
-		create_or_upgrade_to_version ${cluster} '1.0.0'
+		create_or_upgrade_to_version ${cluster} '0.2.0' "${test_dir}/conf/0.2.0_operator.yaml" 'dynamic' "$operator_action"
```

Comment:
why do we need the highest privilege here?

Reply:
in 0.2.0 works only 'dynamic'